### PR TITLE
docs: update readme to reflect stable release

### LINKED ISSUE #8942: [Docs] Readme needs to be updated
README has a few outdated items:

- no longer in BETA
- `RC and stable release` could be removed from Q4?
- `list` states `selection and action list planned Q3` - this has passed
- `sticky-header` states `planned Q3 2017`
- add `badge` to futures?

### PR DESCRIPTION
**Pull request by jelbourn**
Fixes #8942

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,8 +84,9 @@ High level stuff planned for Q4 2017 (October - December):
 | Feature          | Status                              | Docs         | Issue          |
 |------------------|-------------------------------------|--------------|----------------|
 | tree             |      In-progress ([sneak peek][31]) |           -  |  [#3175][3175] |
-| sticky-header    |        In-progress, planned Q3 2017 |           -  |   [#474][0474] |
-| virtual-repeat   |        Not started, planned Q4 2017 |           -  |   [#823][0823] |
+| sticky-header    |        In-progress, planned Q2 2018 |           -  |   [#474][0474] |
+| virtual-repeat   |        Prototyping, planned Q1 2018 |           -  |   [#823][0823] |
+| badge            |        In-progress, planned Q4 2017 |           -  |   [#823][0823] |
```
Comment (TheRedD0t) [inline README.md]: I guess you forgot to edit the linked issue, https://github.com/angular/material2/issues/3058 is the right one.

Comment (jelbourn) [reply README.md]: Good catch, fixed

File: README.md [hunk 2]
```diff
@@ -89,8 +84,9 @@ High level stuff planned for Q4 2017 (October - December):
 | Feature          | Status                              | Docs         | Issue          |
 |------------------|-------------------------------------|--------------|----------------|
 | tree             |      In-progress ([sneak peek][31]) |           -  |  [#3175][3175] |
-| sticky-header    |        In-progress, planned Q3 2017 |           -  |   [#474][0474] |
-| virtual-repeat   |        Not started, planned Q4 2017 |           -  |   [#823][0823] |
+| sticky-header    |        In-progress, planned Q2 2018 |           -  |   [#474][0474] |
+| virtual-repeat   |        Prototyping, planned Q1 2018 |           -  |   [#823][0823] |
+| badge            |        In-progress, planned Q4 2017 |           -  |  [#3085][3058] |
```
Comment (merobal) [inline README.md]: @jelbourn typo: link text is 3085, should be 3058

**Comment by fxck** [comment]
Is badge planned to be done in the past?

**Pull request merged**